errai-config/src/main/java/org/jboss/errai/config/rebind/EnvUtil.java

```diff
@@ -296,11 +296,19 @@ private static void addSerializableTypes(final Set<MetaClass> exposedClasses, fi
       MetaClassFactory
         .getAllCachedClasses()
         .stream()
-        .filter(mc -> filter.apply(mc.getFullyQualifiedName()))
+        .filter(mc -> filter.apply(mc.getFullyQualifiedName()) && validateWildcardSerializable(mc))
         .collect(toCollection(() -> exposedClasses));
     }
   }
 
+  private static boolean validateWildcardSerializable(MetaClass mc) {
+    if(mc.isInterface() || (mc.isAbstract() && !mc.isEnum())) {
```

Comment:
Minor nit-pick, but could you please fix the formatting so there is a space between the `if` and first parenthesis?